pkg/storaged/storage-controls.jsx

```diff
@@ -197,11 +197,11 @@ class StorageOnOff extends React.Component {
         return (
             <StorageControl excuse={this.props.excuse}
                             content={(excuse) => (
-                                <OnOffSwitch state={this.state.promise
+                                <Switch value={this.state.promise
                                     ? this.state.promise_goal_state
                                     : this.props.state}
-                                                 enabled={!excuse && !this.state.promise}
-                                                 onChange={onChange} />
+                                        disabled={excuse || this.state.promise}
```

Comment:
This needs to be a real boolean or React will complain: `Boolean(excuse || )`

Reply:
Thanks, fixed (not pushed yet).